content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -65,72 +120,160 @@ resources:
               secret: YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXoxMjM0NTY=
 ```
 
-Each `resources` array item is a separate config and contains a complete configuration. The
-`resources.resources` field is an array of Kubernetes resource names (`resource` or `resource.group`
-that should be encrypted like Secrets, ConfigMaps, or other resources. 
 
-If custom resources are added to `EncryptionConfiguration` and the cluster version is 1.26 or newer, 
-any newly created custom resources mentioned in the `EncryptionConfiguration` will be encrypted. 
+Each `resources` array item is a separate configuration and contains a full set of definitions for
+how Kubernetes should encrypt data. You can have more than one definition to allow for migration
+between an old and a new way of working, or to support different ways of encryption for different
+API kinds.
+
+Within each top-level item, the inner `resources` field is an array of Kubernetes resource types
+that the API server should encrypt.
+You can specify resource types as a simple type based on its URL representation (for example,
+`secrets` for the [Secret API](/docs/reference/kubernetes-api/config-and-storage-resources/secret-v1/)).
+You can also specify encryption for an API group (for example: `events.k8s.io`)
+that should be encrypted. The `providers` array is an ordered list of the possible encryption
+providers. Only one provider type may be specified per top-level entry
+(for example: you can specify `identity` or `aescbc`, but not both in the same item).
+
+If custom resources are added to `EncryptionConfiguration` and the cluster version is 1.26 or newer,
+any newly created custom resources mentioned in the `EncryptionConfiguration` will be encrypted.
 Any custom resources that existed in etcd prior to that version and configuration will be unencrypted
 until they are next written to storage. This is the same behavior as built-in resources.
 See the [Ensure all secrets are encrypted](#ensure-all-secrets-are-encrypted) section.
 
-The `providers` array is an ordered list of the possible encryption providers to use for the APIs that you listed.
+Within an item, the `providers` array is an ordered list of the possible encryption providers to use
+for the APIs that you listed.
 
-Only one provider type may be specified per entry (`identity` or `aescbc` may be provided,
-but not both in the same item).
-The first provider in the list is used to encrypt resources written into the storage. When reading
-resources from storage, each provider that matches the stored data attempts in order to decrypt the
-data. If no provider can read the stored data due to a mismatch in format or secret key, an error
-is returned which prevents clients from accessing that resource.
+This ordered list can contain more than one provider - the order matters - but you cannot repeat the
```

Comment:
nit: Are these `em` dashes or hyphens?

Reply:
Right now that's a hyphen. I could use an en dash or em dash if that's important.